### ISSUE TITLE
Configuration Errors

### ISSUE THREAD

**Issue by KJlt77** [smartmarmot/DBforBIX#51]
The only way to investigate configuration errors is to analyze DBforBIX log. In case if you need find out if you created all Trapper Items, you need to turn on DEBUG logging mode and grep Zabbix responses. This approach is not good for production and big deployments with multiple DBforBIX proxies.

It would be nice to have some Trapper Item where DBforBIX would report configuration problems.

**Comment by vagabondan** [comment]
Yes, you are absolutely right. I think over it too, but I need fresh view on the realization. Could you provide some details of your vision? 
How should diagnostic items be maintained by users? What type should they have? What should be sent to them?

**Comment by KJlt77** [comment]
Sorry for the delay with the answer.

As I see, there are two major error groups in log:
1. DB connection problems and invalid DB queries.
2. Missing trapper items on Zabbix Server.

DB errors and number of missing trapper items in Zabbix responce are logged to the DBforBIX log, so it would be no problem to collect some statistics about them and pass it to the Zabbix Server.

My vision would be to have separate Error Counter items for each error group (or maybe one is enough) on DBforBIX instance scope. The outcome may be the error count during time interval from the last reporting (every 5, 10 or 30 minutes or whatever). So I could create a trigger to alert about something wrong is happening on DBforBIX instance and then I could dig into it's log.

I'm not sure if it is possible to create those items automatically, but anyway for now DBforBIX is not so obvious tool you can use it without reading documentation and understanding how things are working, so the would be no problem to create some additional items manually.

**Comment by smartmarmot** [comment]
Actually a good idea!
